### PR TITLE
DTFS2-5478 - Fix feedback emails

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issues: 

* emails were not sending with notify template
* email address variable was set even when left blank
* emails were not sending if not logged in
* TFM feedback object in DB was empty

# Changes made:

* added template for TFM feedback email
* fixed TFM feedback DB creation
* Changed variables so email sends
* added N/A string if username is null (not logged in) so email sends
* Renamed email variable so is not populated with submitters email address
* fixed some failing tfm-api tests

### CODE REVIEW

File: trade-finance-manager-api/src/v1/controllers/feedback-controller.js [hunk 1]
```diff
@@ -40,16 +40,20 @@ exports.create = async (req, res) => {
     submittedBy,
   } = modifiedFeedback;
 
+  if (submittedBy.username === null) {
```
Comment (IgnatG) [inline trade-finance-manager-api/src/v1/controllers/feedback-controller.js:43]: same as above

File: portal-api/src/v1/controllers/feedback.controller.js [hunk 2]
```diff
@@ -58,6 +58,10 @@ exports.create = async (req, res) => {
     submittedBy
   } = modifiedFeedback;
 
+  if (submittedBy.username === null) {
+    submittedBy.username = 'N/A - not logged in';
```
Comment (IgnatG) [inline portal-api/src/v1/controllers/feedback.controller.js:62]: is this what James asked for? `N\A - not logged in` ?

Comment (Zainzzkk) [reply portal-api/src/v1/controllers/feedback.controller.js:62]: no, I added this because it breaks the template if passing null

Comment (IgnatG) [reply portal-api/src/v1/controllers/feedback.controller.js:62]: Can we leave it as `N\A` only?

File: portal-api/src/v1/controllers/feedback.controller.js [hunk 3]
```diff
@@ -58,6 +58,10 @@ exports.create = async (req, res) => {
     submittedBy
   } = modifiedFeedback;
 
+  if (submittedBy.username === null) {
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply portal-api/src/v1/controllers/feedback.controller.js:61]: done

**Review by IgnatG — APPROVED** [review]

**Pull request merged**